### DISCUSSION TITLE
amt::hr_overlap for raster / spatRaster objects

### DISCUSSION THREAD

**Discussion by AKuepfer123** [jmsigner/amt#74] [Q&A]
Hi,

I hope this is the correct place to post this question. Is it possible to use the amt::hr_overlap function on objects of raster / spatRaster class somehow; that is to calculate the overlap of the utilization distributions within two rasters of identical extent and resolution ?

Thank you,
Amanda

**Comment by jmsigner** [comment]
Hi Amanda, 

this is unfortunately not implemented at the moment. You could *trick* `amt` by using the internal function `amt:::vol_base()`. If you could detail your workflow a bit more, I am happy to come up with an example. 

All the best, 
Johannes

**Reply by AKuepfer123** [reply]
Hi Johannes,

Thanks very much for your response.

So essentially, I have three rasters of the same spatial extent and
resolution. The values of each raster represent the utilisation
distribution of seabird locations from colony A (raster 1), seabird
locations from colony B (raster 2), and fishing effort (raster 3). The
utilisation distribution values of all rasters have been scaled such that
each raster's utilisation distribution values add up to 1. I want to
calculate the Bhattacharyyan index of the two colonies, as well as the
Bhattacharyyan index of each colony with the fishing effort. To keep things
simple, we can just focus on having two rasters. The reason I mention the
fishery raster is because I cannot just use e.g. kernel density analysis
from the amt or the adehabitatHR packages to calculate the
Bhattacharyyan index.

Does this help?

Thanks again,
Amanda
